locomotion-sdk/package.json

```diff
@@ -40,17 +40,22 @@
     "@react-native-community/datetimepicker": "^6.1.3",
     "react-native-location": "^2.5.0",
     "react-native-mixpanel": "^1.2.5",
-    "react-native-screens": "^3.3.0",
-    "react-native-reanimated": "^2.2.0",
+    "react-native-reanimated": "^2.8.0",
     "@react-native-community/masked-view": "^0.1.11",
-    "react-native-safe-area-context": "^3.2.0",
     "@react-native-picker/picker": "^2.4.1",
     "react-native-modal": "^13.0.1",
     "jwt-decode": "^3.1.2",
     "@stripe/stripe-react-native": "^0.8.0",
-    "react-navigation": "^3.13.0"
+    "react-native-safe-area-context": "^4.2.5",
+    "react-native-screens": "^3.13.1",
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/drawer": "^6.3.0",
+    "@react-navigation/native-stack": "^6.6.2"
   },
   "dependencies": {
+    "@react-navigation/drawer": "^6.3.0",
```

Comment:
need to verfiy its not peer dep
@oriefrati

Reply:
it is, i added them to dependancies also so i could work on the code with types

Reply:
are they native dependancies?